build error : cannot convert 'const bool' to 'boolean' for two_pass_quantize

Build fails roughly halfway through with the following error, using GCC13.   This happens with both release libjxl-0.9.1 and master git version

cmake  -DJXL_HWY_DISABLED_TARGETS_FORCED=OFF -DCXX_FUZZERS_SUPPORTED=NO -DJPEGXL_ENABLE_FUZZERS=NO ..
make
......
Consolidate compiler generated dependencies of target jxl_extras_core-obj
[ 43%] Building CXX object lib/CMakeFiles/jxl_extras_core-obj.dir/extras/dec/jpg.cc.o
/discover/nobackup/ssinno/dl/cespit-1.2/src/libjxl-0.9.1/lib/extras/dec/jpg.cc: In lambda function:
/discover/nobackup/ssinno/dl/cespit-1.2/src/libjxl-0.9.1/lib/extras/dec/jpg.cc:281:42: error: cannot convert ‘const bool’ to ‘boolean’ in assignment
  281 |       cinfo.two_pass_quantize = dparams->two_pass_quant;
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~
      |                                          |
      |                                          const bool
make[2]: *** [lib/CMakeFiles/jxl_extras_core-obj.dir/build.make:244: lib/CMakeFiles/jxl_extras_core-obj.dir/extras/dec/jpg.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1866: lib/CMakeFiles/jxl_extras_core-obj.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
(cespit-1.7b) [ssinno@discover22 build]gcc --version
gcc (GCC) 13.1.0

--- a/lib/extras/dec/jpegli.cc
+++ b/lib/extras/dec/jpegli.cc
@@ -231,7 +231,7 @@ Status DecodeJpeg(const std::vector<uint8_t>& compressed,
     if (dparams.num_colors > 0) {
       cinfo.quantize_colors = TRUE;
       cinfo.desired_number_of_colors = dparams.num_colors;
-      cinfo.two_pass_quantize = dparams.two_pass_quant;
+      cinfo.two_pass_quantize = static_cast<boolean>(dparams.two_pass_quant);
       cinfo.dither_mode = (J_DITHER_MODE)dparams.dither_mode;
     }
 

--- a/lib/extras/dec/jpg.cc
+++ b/lib/extras/dec/jpg.cc
@@ -282,7 +282,7 @@ Status DecodeImageJPG(const Span<const uint8_t> bytes,
     if (dparams && dparams->num_colors > 0) {
       cinfo.quantize_colors = TRUE;
       cinfo.desired_number_of_colors = dparams->num_colors;
-      cinfo.two_pass_quantize = dparams->two_pass_quant;
+      cinfo.two_pass_quantize = static_cast<boolean>(dparams->two_pass_quant);
       cinfo.dither_mode = (J_DITHER_MODE)dparams->dither_mode;
     }
 

